packages/tx/src/1559/constructors.ts

```diff
@@ -0,0 +1,103 @@
+import { RLP } from '@ethereumjs/rlp'
+import { bytesToBigInt, bytesToHex, equalsBytes, validateNoLeadingZeroes } from '@ethereumjs/util'
+
+import { TransactionType } from '../types.js'
+import { txTypeBytes, validateNotArray } from '../util.js'
+
+import { FeeMarketEIP1559Transaction } from './tx.js'
+
+import type { TxOptions } from '../types.js'
+import type { TxData, TxValuesArray } from './tx.js'
+
+/**
+ * Instantiate a transaction from a data dictionary.
+ *
+ * Format: { chainId, nonce, maxPriorityFeePerGas, maxFeePerGas, gasLimit, to, value, data,
+ * accessList, v, r, s }
+ *
+ * Notes:
+ * - `chainId` will be set automatically if not provided
+ * - All parameters are optional and have some basic default values
+ */
+export function create1559FeeMarketTx(txData: TxData, opts: TxOptions = {}) {
+  return new FeeMarketEIP1559Transaction(txData, opts)
+}
+
+/**
+ * Create a transaction from a values array.
```

Comment:
If we're gonna rename the constructor to `*fromBytesArray`, we should be consistent and update the docs too.  Might be better do something like:
```suggestion
 * Create a transaction from a array of byte encoded values ordered according to the devp2p network encoding - format noted below.
```